addons/mail/static/src/components/attachment_image/attachment_image.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+    <t t-name="mail.AttachmentImage" owl="1">
+        <div class="o_AttachmentImage"
+             t-att-class="{
+                    'o-downloadable': attachment and attachment.isDownloadable,
+                    'o-editable': props.isEditable,
+                    'o-isUploading': attachment and attachment.isUploading,
+                    'o-viewable': attachment and attachment.isViewable,
+                    }" t-att-title="attachment ? attachment.displayName : undefined" t-att-data-attachment-local-id="attachment ? attachment.localId : undefined"
```

Comment:
messed up indent here, should be just one level for the object, and then the closing should be on the same level as the opening

Reply:
Avoid using localId, have data-id instead.